### DISCUSSION TITLE
Has the newest version just...broken for anyone else?

### DISCUSSION THREAD

**Discussion by DShip91** [Raytwo/ARCropolis#302] [Issues] [answered]
I can't get the game to boot no matter what I do. It was working just fine last night and this morning but the next time I tried to boot up the game, it just wouldn't. I didn't change anything in my mods folder or my plugins folder so I really didn't know what to do.

- I tried disabling every plugin except for `libarcropolis` and `libnro_hook`. That didn't work.
- I deleted and re-installed the ARCropolis plugin. That didn't work.
- I downgraded to ARCropolis 3.1.1 like the last time one of my plugins broke a new release. Didn't work this time.

Also, every time the game fails to boot, all it tells me is `The software closed because an error occurred.` which is maddening becuause it doesn't generate a crash log which I could have shown someone to try to make sense of what was happening. I'm at a loss.

**Comment by DShip91** [comment]
UPDATE: I think I figured out what happened. One of the mods that I had disabled seems to have glitched out and wasn't being recognized in Windows Explorer which might be what's been causing ARCropolis to fail to boot. I'm going to test the game in a bit to see if it's fixed now.

ETA: Never mind, that wasn't the issue after all. Well, now I've _really_ got nothing.

**Reply by BeetStroganoff** [reply]
Same for me. It is undoubtedly a problem with Arcropolis, since I have verified the integrity of all of my files five consecutive times, just to make absolutely sure it is. I don't even have any mods installed, either! I really don't get how they messed it up so bad.

**Reply by Angstdiscord** [reply]
It's infinitely more likely you didn't remove all your plug-ins than it is Arc causing this issue. And honestly the way you worded this was super rude to the devs.

**Reply by Raytwo** [reply]
If 99% of users can play fine but you can’t, I’d argue that the problem either comes from something you’re doing or you are a very isolated case which means we didn’t « mess up so bad » and just couldn’t have anticipated your very specific problem.

If you can’t even reach the point where you can open the menu on boot, there is something wrong with your install, simple as that.
Maybe don’t use FTP if you do, since apparently your files get corrupted.
Maybe have Windows check your SD.

Especially since as you say, it worked, then suddenly didn’t, like, how does that even happen. I think you’re not telling us everything?
Go to sd:/ultimate/arcropolis/config/ then when you find a directory with a few files, create a file called « debug » (no extension), boot the game again, then post the file called « filesystem_dump.txt » located in sd:/ultimate/arcropolis/ and let’s get this over with.

if the file isn’t even generated, the issue comes from your mods. If you can’t even see the Switch boot animation, the problem comes from your Switch. The fact you can’t even get a crash report to generate definitely implies your install is at fault here.

**Reply by BeetStroganoff** [reply]
I am very sorry for the negativity, I had gotten frustrated from this. It isn't the exact same for me, though, and I should not have insinuated that either, since it hasn't worked for me once. Because of that, there is no /config/ folder either. I don't have any plug-ins, I have a clean, fresh installation of Arcropolis and Skyline, no mods, nothing. 

..also I'm using Ryujinx. Hopefully that doesn't invalidate everything I've said, even though someone else is having the same problem.

**Reply by Raytwo** [reply]
Well, using Ryujinx does make a huge difference indeed lol.

**Comment by DShip91** [comment, marked as answer]
Just to provide an update, I installed ARCropolis onto my SD card directly. I had previously transferred the files over using FileZilla which I thought was the exception to the rule that transferring over FTP can corrupt files but apparently I was wrong about that because after deleting the vast majority of the contents of my `ultimate/mods` folder and doing a fresh install of the core plugins, the game boots just fine now. I'm going to do a bit more testing just to make sure that the issue was just a corrupted install and not one of my mods. I'll have more information to offer once I've finished.

**Reply by BeetStroganoff** [reply]
Aw, rats. If this was the problem, then there's no way I'm getting this emulator stuff fixed. Should've just waited for Yuzu to add modding...

**Reply by Raytwo** [reply]
I don't see how Yuzu would make this work better?

**Reply by BeetStroganoff** [reply]
Well, Yuzu is generally better than Ryujinx, in my opinion. I would figure they can fix whatever is happening faster and better, unless it's a real bug with ARCropolis.

**Reply by Angstdiscord** [reply]
If they can fix it so quickly, why does arc only work on ryujinx and not on Yuzu?

**Reply by BeetStroganoff** [reply]
It's probably not their priority right now..
